packages/init/init.ts

```diff
@@ -0,0 +1,94 @@
+import chalk from "chalk";
+import j from "jscodeshift";
+import path from "path";
+import pEachSeries from "p-each-series";
+
+import astTransform from "@webpack-cli/utils/recursive-parser";
+import propTypes from "@webpack-cli/utils/prop-types";
+import runPrettier from "@webpack-cli/utils/run-prettier";
+
+/**
+ *
+ * Maps back transforms that needs to be run using the configuration
+ * provided.
+ *
+ * @param	{Object} transformObject 	- An Object with all transformations
+ * @param	{Object} config 			- Configuration to transform
+ * @returns {Array} - An array with the transformations to be run
+ */
+
+const mapOptionsToTransform = (config): string[] =>
+	Object.keys(config.webpackOptions)
+		.filter((key: string): boolean => propTypes.has(key));
+
+/**
+ *
+ * Runs the transformations from an object we get from yeoman
+ *
+ * @param	{Object} webpackProperties 	- Configuration to transform
+ * @param	{String} action 			- Action to be done on the given ast
+ * @returns {Promise} - A promise that writes each transform, runs prettier
+ * and writes the file
+ */
+
+export default function runTransform(webpackProperties, action: string): any {
```

Comment:
We should make this an interface

Reply:
Nice yes, but it has to become a class in order to create an interface. Or you talking of `webpackProperties`?